apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -84,7 +84,16 @@
 {{/parse_error}}
 {{#missing_input}}
     {{#label}}<span>{{{label}}}</span>{{/label}}
-    <button type="button" class="btn btn-sm btn-secondary small border" data-placement="auto" data-toggle="popover" data-html="true" title="Missing Input" data-content="There is no submitted value for this field.  This may have happened because the question was changed by course staff after the answer was submitted."> Why <i class="fa fa-question-circle" aria-hidden="true"></i></button>
+    <button
+        type="button"
+        class="badge text-dark badge-missing-input btn btn-sm btn-secondary small border"
```

Comment:
This is a helluva combination of class names 😅 could we get away with a standard button, e.g. `btn btn-xs btn-outline-dark`?

Reply:
I'm also in favor of making this a little simpler. These class names are based on those used for the invalid input message: https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/elements/pl-string-input/pl-string-input.mustache#L74

Should we make both of these use the same class names and only change the color of the text (i.e. `text-dark` vs `text-danger`)?

Reply:
Why `text-dark` and such instead of just `btn-*` with an appropriate color? What do we gain from more complicated and non-standard classes? FWIW, my suggestion applies to the invalid input popover buttons as well.

Reply:
Ohhh, when I was playing with this, leaving off the `text-dark` left the text white, which made it unreadable. I'll play around with this first and see how it looks with a more minimal class list.

Reply:
If you run into trouble, share the class list you were using; you definitely shouldn't be ending up with white text, so you might have another extraneous class in there.

Reply:
Updated the class list in the latest commit: https://github.com/PrairieLearn/PrairieLearn/pull/10701/commits/542af0986ce39fb5b313c7c0e20d26d925e51159

Screenshot:
![image](https://github.com/user-attachments/assets/b55e22e7-b5bf-49af-b207-c493d50d5224)

I think the buttons are maybe a little too small? But not really a big deal in my opinion, the other button sizes are too big.

Reply:
Ahhh that's a good suggestion actually. I was thinking that we wanted the buttons to be as similar as possible besides the text color, but switching the outline maybe looks more natural. I'll try that and post a screenshot.

Reply:
@nwalters512 changed to match your suggestion, I think it looks better this way:

![image](https://github.com/user-attachments/assets/941842c9-e869-4a2c-8055-830253ada3e0)

Not sure whether this totally resolves the issue of students not realizing this is clickable, but it at least seems more clickable than before?

Reply:
I think these styles look great! Re: making it more obviously clickable, we discussed on #10765 the possibility of changing the exclamation icon to a question mark. We generally uses question marks to mean "click/hover for more information", so doing the same here might help.

Reply:
Normally I'd agree, but I think that the exclamation mark is pretty ingrained into the elements, to the point that changing this might confuse students. We can discuss during the upcoming meeting however.

Reply:
From meeting: Change all red triangles to question marks.